models/base.py

```diff
@@ -260,6 +264,7 @@ def set_column_sizes_on_view(self, table_view):
             # is a workaround for that case.
             # TODO: the graph view should use ``model()`` instead of setting
             # to model.
+            logger.exception("TODO: fix the graph view")
             if table_view.model is None:
                 raise RuntimeError("No model set on view.")
 
```

Comment:
nu is inderdaad duidelijk waar de RuntimeError vandaan komt. De fix gaan we fixen in een andere branch

Version result file has been created with: b'1.4.16-1'
Version gridadmin file has been created with: 1.4.15-1
ThreeDiToolbox.utils.utils CRITICAL Unknown agg method: current
ThreeDiToolbox.utils.utils CRITICAL Unknown agg method: current
ThreeDiToolbox.utils.utils CRITICAL Unknown agg method: current
ThreeDiToolbox.utils.utils CRITICAL Unknown agg method: current
ThreeDiToolbox.models.base ERROR TODO: fix the graph view
Traceback (most recent call last):
  File "/home/nens/.local/share/QGIS/QGIS3/profiles/default/python/plugins/ThreeDiToolbox/models/base.py", line 260, in set_column_sizes_on_view
    if table_view.model() is None:
TypeError: 'LocationTimeseriesModel' object is not callable
ThreeDiToolbox.models.base ERROR TODO: fix the graph view
Traceback (most recent call last):
  File "/home/nens/.local/share/QGIS/QGIS3/profiles/default/python/plugins/ThreeDiToolbox/models/base.py", line 260, in set_column_sizes_on_view
    if table_view.model() is None:
TypeError: 'LocationTimeseriesModel' object is not callable

Reply:
See #382